exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/_log_exporter/__init__.py

```diff
@@ -121,8 +121,9 @@ def __init__(
         )
         self._compression = compression or _compression_from_env()
         self._session = session or requests.Session()
-        self._session.headers.update(self._headers)
         self._session.headers.update(_OTLP_HTTP_HEADERS)
+        # let users override our defaults
+        self._session.headers.update(self._headers)
```

Comment:
The linked spec PR currently says `Typically an exporter would *prepend* the given identifier to its own.` but I don't think that matters much.